jstests/aggregation/bugs/server33149.js

```diff
@@ -0,0 +1,20 @@
+(function() {
+    "use strict";
+
+    // For arrayEq.
+    load("jstests/aggregation/extras/utils.js");
+
+    let indexDB = db.getSiblingDB("create_indexess_db");
+    assert.commandWorked(indexDB.dropDatabase());
+
+    // Create an index with key: {x : 1}, name : "x_1
+    assert.commandWorked(indexDB.runCommand({ createIndexes : "test", indexes : [ { key : { x : 1 }, name : "x_1" } ] }));
+
+    // Create a duplicate index will not result an error
+    assert.commandWorked(indexDB.runCommand({ createIndexes : "test", indexes : [ { key : { x : 1 }, name : "x_1"} ] }), ErrorCodes.IndexAlreadyExists, "Identical index already exists ");
```

Comment:
It looks like you're passing extra options to 'commandWorked' here - they are probably being ignored? If so, just delete them.
https://github.com/mongodb/mongo/blob/b7b58a38dd34ac09ed56eb1f306ea6d64f8b2d88/src/mongo/shell/assert.js#L588